ocd/src/probe/flash/builder.rs

```diff
@@ -267,14 +267,12 @@ impl<'a> FlashBuilder<'a> {
             } else {
                 self.chip_erase_program(&mut flash, &sectors)?;
             };
-        } else {
-            if flash.double_buffering_supported() && self.enable_double_buffering {
-                self.sector_erase_program_double_buffer(&mut flash, &mut sectors)?;
-            } else {
-                // WORKING: We debug this atm.
-                self.sector_erase_program(&mut flash, &sectors)?;
-            };
-        }
+        } else if flash.double_buffering_supported() && self.enable_double_buffering {
+    self.sector_erase_program_double_buffer(&mut flash, &mut sectors)?;
+} else {
+    // WORKING: We debug this atm.
+    self.sector_erase_program(&mut flash, &sectors)?;
+}
```

Comment:
This is now pretty badly formatted

Reply:
Sorry, how is this badly formatted? :/

Reply:
`cargo fix` has removed the indentation

Reply:
Ah, can't see that in the snippet :D thx